x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/errors/errors_page.spec.ts

```diff
@@ -110,3 +110,66 @@ describe('Errors page', () => {
     });
   });
 });
+
+describe('Check detailed statistics API with multiple errors', () => {
+  before(async () => {
+    cy.loginAsViewerUser();
+    await synthtrace.index(
+      generateMultipleErrorsData({
+        from: new Date(start).getTime(),
+        to: new Date(end).getTime(),
+      })
+    );
+  });
+
+  after(async () => {
+    await synthtrace.clean();
+  });
+
+  it('calls detailed API with visible items only', () => {
+    cy.intercept(
+      'GET',
+      '/internal/apm/services/opbeans-java/errors/groups/main_statistics?*'
+    ).as('errorsMainStatistics');
+    cy.intercept(
+      'POST',
+      '/internal/apm/services/opbeans-java/errors/groups/detailed_statistics?*'
+    ).as('errorsDetailedStatistics');
+    cy.visit(`${javaServiceErrorsPageHref}&pageSize=10`);
+    cy.wait('@errorsMainStatistics');
+    cy.get('.euiPagination__list').children().should('have.length', 5);
+    cy.wait('@errorsDetailedStatistics').then((payload) => {
+      expect(payload.request.body.groupIds).eql(
+        JSON.stringify([
+          '0000000000000000000000000Error 0',
+          '0000000000000000000000000Error 1',
+          '0000000000000000000000000Error 2',
+          '0000000000000000000000000Error 3',
+          '0000000000000000000000000Error 4',
+          '0000000000000000000000000Error 5',
+          '0000000000000000000000000Error 6',
+          '0000000000000000000000000Error 7',
+          '0000000000000000000000000Error 8',
+          '0000000000000000000000000Error 9',
+        ])
+      );
+    });
+    cy.get('[data-test-subj="pagination-button-1"]').click();
+    cy.wait('@errorsDetailedStatistics').then((payload) => {
+      expect(payload.request.body.groupIds).eql(
+        JSON.stringify([
+          '000000000000000000000000Error 10',
+          '000000000000000000000000Error 11',
+          '000000000000000000000000Error 12',
+          '000000000000000000000000Error 13',
+          '000000000000000000000000Error 14',
+          '000000000000000000000000Error 15',
+          '000000000000000000000000Error 16',
+          '000000000000000000000000Error 17',
+          '000000000000000000000000Error 18',
+          '000000000000000000000000Error 19',
+        ])
```

Comment:
Same as above. I'd expect this to be something like:
```suggestion
      expect(payload.request.body.groupIds.length).eql(NUMBER_OF_VISIBLE_ERROR_GROUPS)
```

Reply:
check the comment above.